### ISSUE TITLE
[Android] AtmosphereTestCase that use MultipleSources

### ISSUE THREAD

**Issue by Buzzfease** [mousebird/WhirlyGlobe#1218]
Hi, guys
The code in TestCases/AtmosphereTestCase.java shows "The night images have gone missing". but when i changed the night image remote source to an avalible remote source and set the 
`boolean UseMultipleSources = true;`
I supposed to shows daytime layer image on the sunny side and night-time layer image on the dark side ,but i got nothing changed, can anyone confirm this function works normally?
Thank you for any help.

**Comment by mousebird** [comment]
Yup, does seem to be broken.  I'll take a look at it in 3.0 at some point.
